x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/type/DataType.java

```diff
@@ -126,9 +127,9 @@
 
 
     /**
-     * Elasticsearch type name
+     * Lowercase type name
```

Comment:
I am not sure "Lowercase" is the right description for this field, I mean it is a true statement, but doesn't help much with the purpose of the field.

Reply:
+1. 
The purpose of a comment is to explain the non-obvious:
```
/** an integer */ vs /** array index */
public int i;
```